phpstan.dist.baseline.neon

```diff
@@ -500,21 +485,6 @@ parameters:
 			count: 1
 			path: app/code/core/Mage/Adminhtml/controllers/Report/StatisticsController.php
 
-		-
-			message: "#^Method Mage_Adminhtml_Model_Sales_Order_Create\\:\\:moveQuoteItem\\(\\) invoked with 2 parameters, 3 required\\.$#"
```

Comment:
Was it intentional to remove all these errors from the baseline? Cause this one for example wasn't resolved and shows up on the "Files" tab

Reply:
the thing is that phpstan kept telling me "these errors in the baseline are ignored and not necessary" so I kept removing, but there's for sure something wrong with my phpstan (although I've no idea what)